pkg/minikube/machine/cache_images.go

```diff
@@ -162,6 +189,18 @@ func getSrcRef(image string) (types.ImageReference, error) {
 }
 
 func getDstRef(image, dst string) (types.ImageReference, error) {
```

Comment:
Could you maybe add a few unit tests here for the new behavior?

Reply:
Ok. I will add the unit tests from now.